platform/lib/routers/go.js

```diff
@@ -81,6 +81,7 @@ function initGoLinks(config) {
       });
     } else {
       simple[key] = value.url;
+      simple[`${key}/`] = value.url;
```

Comment:
what if the key already has a trailing slash?

Reply:
These are static strings, it would be `/foo/` and `/foo//`. Handling it felt like the complexity outweighed the use since there are no uses of it currently. It would allow for `/foo/` and `/foo//`. 

That being said, I do something like

```js 
} else {
  key =  key.replace(/\/?$/, '/?');
  simple[key] = value.url;
}
```

which would force an optional trailing slash (e.g. `/?`) on all `simple` routes

Reply:
I think it'd be good to consistently support both schemes with and without trailing slash. Stripping trailing slashes when loading the input and when handling a request (like in your first version) might be the easiest way to do this.

Reply:
> I think it'd be good to consistently support both schemes with and without trailing slash.

makes sense

pushed an update, wdyt